repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py

```diff
@@ -286,6 +287,27 @@ def report_skipped_packages(message, packages):
         api.show_message(summary)
 
 
+def add_output_pkgs_to_transaction_conf(transaction_configuration, events):
+    """
+    Add more packages for removal to transaction configuration if they can be derived as outputs of PES events.
+
+    Output packages from an event are added to packages for removal only if all input packages are already there.
+
+    :param events: List of Event tuples, where each event contains event type and input/output pkgs
+    :param transaction_configuration: RpmTransactionTasks model instance with pkgs to install, keep and remove based
+                                      on the user configuration files
+    """
+    message = 'Marking packages for removal as a result of events:\n'
+
+    for event in events:
+        if event.action in ('Split', 'Merged'):
```

Comment:
What about `Replaced` and `Renamed` events?

Reply:
After discussing this with others, it makes sense to apply this PR to `Replaced` and `Renamed` too. I'll add those, thanks for the remark.